src/community/__stories__/blocks-story.tsx

```diff
@@ -144,9 +145,8 @@ export const BlockHighlightedValue: StoryComponent<HighlightedValueBlockArgs> =
                     tag={<Tag type={tag}>{`${tag}`}</Tag>}
                     title={title}
                     description={description}
-                    value={value}
+                    mainHeading={{value, text}}
                     secondaryValue={secondaryValue}
```

Comment:
not used, right?

Reply:
in this case the value and text are being used to pass the object with their information in the mainHeadling